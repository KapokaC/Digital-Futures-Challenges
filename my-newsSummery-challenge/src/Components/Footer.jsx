const Footer = () => {
    return (

        <footer className="footer">

            <div className="container">
                <p>Â© DF - Charity Cleopatra Kapoka 2022</p>
            </div>

        </footer>
    )
}

export default Footer;