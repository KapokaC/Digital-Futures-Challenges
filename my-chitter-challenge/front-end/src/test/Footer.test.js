import { render } from '@testing-library/react';
import Footer from '../components/home/footer.jsx';
import { MemoryRouter } from 'react-router-dom';


describe('Footer tests', () => {
    test(`Footer matches snapshot`, () => {

        expect(render(<MemoryRouter><Footer /></MemoryRouter>)).toMatchSnapshot();
    });
})