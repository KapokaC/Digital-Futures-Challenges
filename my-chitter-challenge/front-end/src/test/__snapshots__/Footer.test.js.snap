// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer tests Footer matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="container"
      >
        <footer
          class="text-center text-white"
          style="background-color: rgb(10, 66, 117);"
        >
          <div
            class="text-center p-3"
            style="background-color: black;"
          >
            © 2020 Copyright:
            <a
              class="text-white"
              href="https://digitalfutures.com/"
            >
              Charity Kapoka 2022
            </a>
          </div>
        </footer>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="container"
    >
      <footer
        class="text-center text-white"
        style="background-color: rgb(10, 66, 117);"
      >
        <div
          class="text-center p-3"
          style="background-color: black;"
        >
          © 2020 Copyright:
          <a
            class="text-white"
            href="https://digitalfutures.com/"
          >
            Charity Kapoka 2022
          </a>
        </div>
      </footer>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
